src/main/scala/org/ergoplatform/settings/LaunchParameters.scala

```diff
@@ -2,14 +2,18 @@ package org.ergoplatform.settings
 
 import org.ergoplatform.settings.Parameters._
 
-object LaunchParameters extends Parameters(height = 0, parametersTable = Map(
-  StorageFeeFactorIncrease -> Parameters.StorageFeeFactorDefault,
-  MinValuePerByteIncrease -> Parameters.MinValuePerByteDefault,
-  TokenAccessCostIncrease -> Parameters.TokenAccessCostDefault,
-  InputCostIncrease -> Parameters.InputCostDefault,
-  DataInputCostIncrease -> Parameters.DataInputCostDefault,
-  OutputCostIncrease -> Parameters.OutputCostDefault,
-  MaxBlockSizeIncrease -> 512 * 1024,
-  MaxBlockCostIncrease -> 1000000,
-  BlockVersion -> 1
-))
+object LaunchParameters extends Parameters(
+  height = 0,
+  parametersTable = Map(
```

Comment:
It would be more readable to define `parametersTable` outside the `Parameters` constructor.